pack.yaml

```diff
@@ -5,6 +5,6 @@ runner_type: "python-script"
 description: Terraform integrations
 keywords:
     - terraform
-version: 0.2.6
+version: 0.2.7
```

Comment:
According to https://semver.org/, I think the version should be `0.3.0` since it's not a bugfix, but rather a change

Reply:
I almost typed that same thing @blag said from my phone

Reply:
ok cool, I've updated the pr.